README.md

```diff
@@ -42,15 +42,15 @@ For failed test runs you can expand each failed test and view more details about
 <kbd><img src="./docs/failed_tests.png"></img></kbd>
 
```

Comment:
I can't make a comment on the correct line, but in the readme example you'll need to update the version of this action to v2.0.0.   When you commit that change please add `+semver:major` to the commit message so the increment-version-on-merge workflow knows how to increment this appropriately.